accounts/abi/abi_test.go

```diff
@@ -60,7 +60,7 @@ const jsondata2 = `
 ]`
```

Comment:
Why don't you add test cases introduced in https://github.com/ethereum/go-ethereum/pull/18406 ?

Reply:
Thanks! Added https://github.com/klaytn/klaytn/pull/792/commits/221460711cccb17149f88f4fb8f7d190a693473b